Radar/Radar/Radar/Plane.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#ifndef PLANE_H
+#define PLANE_H
+
+#include <cstdint> // std::uint32_t
+#include <utility> // std::pair
+
+struct Point
+{
+public:
+	std::uint32_t s_planeCoordinateX;
+	std::uint32_t s_planeCoordinateY;
+
+public:
+	Point(std::uint32_t planeCoordinateX, std::uint32_t planeCoordinateY);
+
+	friend std::ostream& operator<<(std::ostream &out, const Point &planePoint);
+};
+
+struct Speed
+{
+public:
+	int s_planeSpeedX;
+	int s_planeSpeedY;
+
+public:
+	Speed(int planeSpeedX, int planeSpeedY);
+
+	friend std::ostream& operator<<(std::ostream &out, const Speed &planeSpeed);
+};
+
+class Plane 
+{
+private:
+	std::uint32_t c_planeId;
```

Comment:
The Id is not allowed to change 
```suggestion
	const std::uint32_t c_planeId;
```